ethereum_history_api/circuits/lib/src/verifiers/header.nr

```diff
@@ -46,47 +47,31 @@ pub fn verify_header(
         expected_header_fields_count == header_rlp_list.num_fields, "number of header RLP fields does not match"
     );
 
-    // We use u40 for block number because even if block time is 1s - it would take 34865 years to reach u40
```

Comment:
Why do we delete this comment? And what's u40?

Reply:
u40 is an unsigned type that does not exist in Noir any more. Now we have u8, u32, u64 and that's it. In good ol' days we've had u1, u8, u16, u24 etc with a step of 8 up to u128